[2023-10-02T09:58:09.338+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Load-Green-Taxi-Data-Web-To-GCS-To-Postgres.read_postgres scheduled__2020-11-02T06:00:00+00:00 [queued]>
[2023-10-02T09:58:09.349+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Load-Green-Taxi-Data-Web-To-GCS-To-Postgres.read_postgres scheduled__2020-11-02T06:00:00+00:00 [queued]>
[2023-10-02T09:58:09.350+0000] {taskinstance.py:1308} INFO - Starting attempt 2 of 3
[2023-10-02T09:58:09.674+0000] {taskinstance.py:1327} INFO - Executing <Task(PostgresOperator): read_postgres> on 2020-11-02 06:00:00+00:00
[2023-10-02T09:58:09.688+0000] {standard_task_runner.py:57} INFO - Started process 150 to run task
[2023-10-02T09:58:09.693+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'Load-Green-Taxi-Data-Web-To-GCS-To-Postgres', 'read_postgres', 'scheduled__2020-11-02T06:00:00+00:00', '--job-id', '745', '--raw', '--subdir', 'DAGS_FOLDER/web_gcs_pg.py', '--cfg-path', '/tmp/tmpqr04_dhl']
[2023-10-02T09:58:09.696+0000] {standard_task_runner.py:85} INFO - Job 745: Subtask read_postgres
[2023-10-02T09:58:09.837+0000] {task_command.py:410} INFO - Running <TaskInstance: Load-Green-Taxi-Data-Web-To-GCS-To-Postgres.read_postgres scheduled__2020-11-02T06:00:00+00:00 [running]> on host 537710aea1a2
[2023-10-02T09:58:10.178+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='Load-Green-Taxi-Data-Web-To-GCS-To-Postgres' AIRFLOW_CTX_TASK_ID='read_postgres' AIRFLOW_CTX_EXECUTION_DATE='2020-11-02T06:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2020-11-02T06:00:00+00:00'
[2023-10-02T09:58:10.179+0000] {sql.py:265} INFO - Executing: SELECT * FROM green_taxi  5;
[2023-10-02T09:58:10.189+0000] {base.py:73} INFO - Using connection ID 'my_postgres_conn' for task execution.
[2023-10-02T09:58:10.442+0000] {base.py:73} INFO - Using connection ID 'my_postgres_conn' for task execution.
[2023-10-02T09:58:10.446+0000] {sql.py:375} INFO - Running statement: SELECT * FROM green_taxi  5;, parameters: None
[2023-10-02T09:58:10.447+0000] {taskinstance.py:1824} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/common/sql/operators/sql.py", line 277, in execute
    **extra_kwargs,
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/common/sql/hooks/sql.py", line 349, in run
    self._run_command(cur, sql_statement, parameters)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/common/sql/hooks/sql.py", line 380, in _run_command
    cur.execute(sql_statement)
psycopg2.errors.SyntaxError: syntax error at or near "5"
LINE 1: SELECT * FROM green_taxi  5;
                                  ^

[2023-10-02T09:58:10.455+0000] {taskinstance.py:1350} INFO - Marking task as UP_FOR_RETRY. dag_id=Load-Green-Taxi-Data-Web-To-GCS-To-Postgres, task_id=read_postgres, execution_date=20201102T060000, start_date=20231002T095809, end_date=20231002T095810
[2023-10-02T09:58:10.954+0000] {standard_task_runner.py:109} ERROR - Failed to execute job 745 for task read_postgres (syntax error at or near "5"
LINE 1: SELECT * FROM green_taxi  5;
                                  ^
; 150)
[2023-10-02T09:58:10.988+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 1
[2023-10-02T09:58:11.024+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
